app/views/form/_check_answers_breadcrumb.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="govuk-breadcrumbs">
+  <ol class="govuk-breadcrumbs__list">
+    <li class="govuk-breadcrumbs__list-item">
+      <a class="govuk-breadcrumbs__link" href="/logs">Logs</a>
+    </li>
+    <li class="govuk-breadcrumbs__list-item">
+      <a class="govuk-breadcrumbs__link" href=" /logs/<%= @case_log.id %>">Log</a>
```

Comment:
Could possibly DRY this into a single breadcrumbs partial with a variable?

Reply:
There’s also the GOV.UK Components helper we can use for breadcrumbs: https://govuk-components.netlify.app/components/breadcrumbs/

Reply:
That is actually quite perfect! Changed that now, let me know if it looks okay!